static/styles/custom/vss.css

```diff
@@ -1,5 +1,5 @@
-.highlight .hll { background-color: #ffffcc }
-.highlight  { background: #ffffff; }
+.highlight .hll { background-color: #080808 }
+.highlight  { background: #030303; color:aliceblue; }
```

Comment:
What you have done here is to modify the *default* highlighting so it has the black background. It would be better to only modify the background of `.language-terminal .highlight`. That way, we could explicitly enable the terminal-look using <code>```terminal</code>.